dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala

```diff
@@ -285,18 +279,43 @@ object LagomPlugin extends AutoPlugin {
     scalaVersion := "2.11.7",
     libraryDependencies += LagomImport.component("lagom-cassandra-server"),
     lagomCassandraStart in ThisBuild := startCassandraServerTask.value,
-    lagomCassandraStop in ThisBuild := Servers.CassandraServer.tryStop(state.value.log)
+    lagomCassandraStop in ThisBuild := Servers.CassandraServer.tryStop(new SbtLoggerProxy(state.value.log))
   ))
 
+  private val projectPortMap = AttributeKey[Map[ProjectName, Port]]("lagomProjectPortMap")
+  private val defaultPortRange = PortRange(20000, 30000)
```

Comment:
Shouldn't this be the same as in https://github.com/lagom/lagom/pull/137/files#diff-d142c6e1820bd14ed4be0ccdbb3444e7R6 (I guess you can just take the opportunity to make the change and fix #101